src/cli/main.rs

```diff
@@ -16,6 +16,16 @@ use clap::Parser;
 
 use super::subcommands::Subcommand;
 
+#[macro_export]
```

Comment:
Why is this a macro? It looks like it could be a simple function, or am I missing something?

Reply:
Ok removing it :)